omnibus/config/projects/iot-agent.rb

```diff
@@ -65,6 +65,8 @@
   # creates required build directories
   dependency 'preparation'
 
+  dependency "systemd" if linux_target?
```

Comment:
Should we perhaps move this to the datadog-iot-agent.rb software definition instead.

Reply:
That seems more sensible indeed, done!